src/portal-components/DetailsPage.tsx

```diff
@@ -438,6 +438,15 @@ function isReactFragment(variableToInspect: any): boolean {
   return variableToInspect === React.Fragment
 }
 
+const getSynapseComponent = (el: RowSynapseConfig, queryResultBundle?: QueryResultBundle) => {
```

Comment:
Pulled out existing code into a reusable function, that will also be called when processing toggleConfigs